cpp_test_suite/new_tests/cxx_nan_inf_in_prop.cpp

```diff
@@ -205,12 +214,12 @@ class NanInfInPropSuite: public CxxTest::TestSuite
 		// Extract as float:
 		Tango::DevFloat float_val;
 		TS_ASSERT(db_data_r[0] >> float_val);
-		TS_ASSERT(std::isnan(float_val));
+		TS_ASSERT(isnan(float_val));
 
 		// Extract as double:
 		Tango::DevDouble double_val;
 		TS_ASSERT(db_data_r[0] >> double_val);
-		TS_ASSERT(std::isnan(double_val));
+		TS_ASSERT(isnan(double_val));
```

Comment:
This change is breaking the travis CI build with the following error:

/home/tango/src/cpp_test_suite/cxxtest/include/../../new_tests/cxx_nan_inf_in_prop.cpp:222:29: error: call of overloaded 'isnan(Tango::DevDouble&)' is ambiguous

Reply:
since it's on the test part, I can revert and we will see test after

Reply:
Nice try! ;-) but this line was not the only one like that... There are many errors similar to this one.

Reply:
I changed all in the file ;)